packages/loctool/.gitignore

```diff
@@ -1,10 +0,0 @@
-jsdoc
-java/bin
-build.properties
```

Comment:
These two need to stay

Reply:
What are they for?

Reply:
There is a little bit of java in the project to provide an ilib-like API to some Java apps. This is no longer used, as the original company I wrote this for, HealthTap, no longer does localization. But, it is there for now and should be cleaned up later after we verify my assertion about HealthTap